### PR TITLE
prevent index from growing infinitely

### PR DESCRIPTION
**Pull request by noamokman**
Hi there!

it seems that the variable `i` just keeps growing.
This code works the same but sets `i` back to 0 every cycle.

Thanks!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,7 +8,11 @@ module.exports = function () {
 	var i = 0;
 
 	return function () {
-		return frames[i++ % frames.length];
```
Comment (sindresorhus) [inline index.js]: Could just be `return frames[i = ++i % frames.length];`, right?

**Comment by sindresorhus** [comment]
Tests are failing.

**Comment by noamokman** [comment]
@sindresorhus opps :smile: tests pass now and I made the changes as you said :+1:

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request closed without merging**